README.md

```diff
@@ -16,7 +16,7 @@ Lightweight (169MB) [Crashplan](http://www.crashplan.com) docker container.
 docker run -d \
   --name crashplan \
   -h $HOSTNAME \
-  -e TZ \
+  -e TZ=America/Los_Angeles \
```

Comment:
If you want to force the TZ variable export it's value before calling the docker run command.